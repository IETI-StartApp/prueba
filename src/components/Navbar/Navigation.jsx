import React, {useState} from 'react';
import {Hidden} from '@material-ui/core/';
import Navbar from './index';
import MenuDrawer from './MenuDrawer';
import {useStyles} from './styles';

export default function Navigation() {

    const classes = useStyles();

    const [menuOpen, setMenuOpen] = useState(false);

    const handleDrawerToggle = () => {
        setMenuOpen(!menuOpen);
    };

    return (
        <div className={classes.root}>
            <Navbar
                handleDrawerToggle={handleDrawerToggle}/>
            <Hidden>
                <MenuDrawer
                    open={menuOpen}
                    onClose={handleDrawerToggle}/>
            </Hidden>
        </div>
    );
}
