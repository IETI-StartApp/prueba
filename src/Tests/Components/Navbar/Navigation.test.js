import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import React from 'react';
import Navigation from '../../../components/Navbar/Navigation';

describe('Testing navigation', () => {

    const wrapper = shallow(<Navigation />)

    test('should load navigation ', () => {
        expect(wrapper).toMatchSnapshot();
    })
})
