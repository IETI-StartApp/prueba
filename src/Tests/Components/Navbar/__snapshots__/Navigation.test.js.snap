// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing navigation should load navigation  1`] = `
<div>
  <Navbar
    handleDrawerToggle={[Function]}
  />
  <Hidden>
    <MenuDrawer
      onClose={[Function]}
      open={false}
    />
  </Hidden>
</div>
`;
